### DISCUSSION TITLE
Committing non-lockfile files

### DISCUSSION THREAD

**Discussion by dovidgef** [renovatebot/renovate#10573] [Request Help]
I have a package that generates configuration files on install.
It appears to me that by default Renovate only commits the lockfile related files.

How can I configure Renovate to commit any changed files for a specific package?

Thanks for your help.

**Comment by rarkins** [comment]
If you are running self-hosted then you can use `postUpgradeTasks` and commit arbitrary files using `fileFilters`

**Reply by dovidgef** [reply]
We are running using your Github App, is there any way to get it working there?

Thanks

**Reply by rarkins** [reply]
Not currently
